workshop/exercise-2/README.md

```diff
@@ -14,21 +14,52 @@ The application for this lab is a simple node.js with express app which returns
 	cd fib-knative
 	```
 
+## Set up a Private Container Registry and Obtain Credentials
+In this lab, we'll use the IBM Container Registry to host our docker images since you already have access to this through your IBM Cloud Account. IBM Container Registry enables you to store and distribute container images in a fully managed private registry.
+
+1. Install the container registry plugin:
+
+  ```
+  ibmcloud plugin install container-registry
+  ```
+2. Choose a name for your first namespace, and create it. 
+
+  ```
+  ibmcloud cr namespace-add <my_namespace>
+  ```
+3. Create a token. This token is a non-expiring token with read and write access to all namespaces the region.
```

Comment:
Add something about why they're doing this - e.g. `We need to do this so that the automated build processes we'll be setting up can access your images`.